src/7-to-8/major-changes/platforms.rst

```diff
@@ -6,46 +6,11 @@ Platforms
 .. admonition:: Does This Change Affect Me?
    :class: tip
 
-   .. cylc-scope:: flow.cylc
+   Cylc platforms are a new feature which replace the task ``job`` and
+   ``remote`` configuration sections:
 
-   Platforms replace the deprecated :cylc:conf:`[runtime][<namespace>][job]`
-   and :cylc:conf:`[runtime][<namespace>][remote]`
-   sections:
-
-   .. code-block:: cylc
-
-      [runtime]
-          [[foo]]
-              [[[job]]]
-                  batch system = slurm
-              [[[remote]]]
-                  host = my_supercomputer
-
-   .. cylc-scope::
-
-   Read this section if your workflow's jobs run on a remote computer or if
-   you see the following warning on running ``cylc validate``:
-
-   .. code-block:: console
-
-      WARNING - deprecated settings found (please replace with [runtime][foo]platform):
-          [runtime][foo][remote]host
-          [runtime][foo][job]batch system
-
-   If you currently use the ``rose host-select`` utility or a similar host
```

Comment:
I think it's worth keeping this paragraph?

Reply:
This is covered by `[runtime][<namespace>][remote]` (note that `rose host-select` can only work when configured by this).